tools/wasme/cli/operator/api/wasme/v1/filter_deployment.proto

```diff
@@ -48,6 +48,11 @@ message FilterSpec {
 
     // custom options if pulling from private / custom repositories
     ImagePullOptions imagePullOptions = 5;
+
+    // a class of configurations based on the traffic flow direction
+    // and workload type.
+    // defaults to `inbound`.
```

Comment:
Might be worth linking to the Istio docs on PatchContext here so that folks looking at our docs have some context:
```suggestion
    // a class of configurations based on the traffic flow direction
    // and workload type.
    // defaults to `inbound`.
    // See https://istio.io/latest/docs/reference/config/networking/envoy-filter/#EnvoyFilter-PatchContext for more details.
```